### ISSUE TITLE
Problem with seed in Saltelli.sample

### ISSUE THREAD

**Issue by jmrk87** [SALib/SALib#334]
I noticed a possible bug in Saltelli.sample function.
In particular, the seed option does not seem to work properly (Python 3.6, Ubuntu).
I attached the following script:

```python
N=1
problem = {'num_vars': 3,
           'names': ['x1','x2','x3'],
           'bounds': [
               [0,1],
               [0,1],
               [0,1]
           ]
           }
from SALib.sample import saltelli
Sample = saltelli.sample(problem, N, calc_second_order=False, seed=None)

print(Sample)
```

As can be easily verified, the sampling provides always the same results (Sample) with any "seed" setting.

**Comment by ConnectedSystems** [comment]
Hi @jmrk87 

Thanks for raising this issue.

As you probably know, the Saltelli method is an extended Sobol sampler, so it uses the deterministic Sobol sequence.

If you want different samples, use the `skip_values` parameter (the default is 1000):

```python
>>> sample = saltelli.sample(problem, N, calc_second_order=False, skip_values=5)
>>> print(sample)

[[0.875 0.875 0.125]
 [0.375 0.875 0.125]
 [0.875 0.875 0.125]
 [0.875 0.875 0.625]
 [0.375 0.875 0.625]]

>>> sample = saltelli.sample(problem, N, calc_second_order=False, skip_values=10)
>>> print(sample)

[[0.9375 0.0625 0.6875]
 [0.1875 0.0625 0.6875]
 [0.9375 0.3125 0.6875]
 [0.9375 0.0625 0.5625]
 [0.1875 0.3125 0.5625]]
```

I added the `seed` parameter as an option as a matter of convenience when creating the command-line interface (it made implementation consistent across all methods). 

As you point out, it makes less sense for the Sobol sequence and does not actually have an effect.
I did not recognize at the time this issue might occur.

@jdherman is it worth the possible confusion to change `skip_values` to `seed`? Or perhaps I handle this as a special case for the CLI and remove the `seed` option.

Might be worthwhile to add more information to the docstring around what the (suggested) minimum and maximum number of points to skip is.

**Comment by jdherman** [comment]
Yea this is a special case where `seed` has no meaning. Changing the name would cause a lot of confusion. It's a good point, and probably calls for both (1) better documentation of `skip_values`, and (2) throw a warning if `seed` is passed into the Saltelli sampling method.

**Comment by ConnectedSystems** [comment]
Addressed in #340 - will be available from v1.4
